src/tui.rs

```diff
@@ -403,6 +409,18 @@ impl ToipeTui {
         //   - won't hang there waiting for user to type space
         lines.push(Text::from(line.join(" ")).with_faint());
 
+        if lines.len() + self.bottom_lines_len + 1 >= terminal_height.into() {
+            return Err(ToipeError::from(format!(
+                "toipe requires atleast {} lines in your terminal",
```

Comment:
nitpick: the message can better written and would be nice to display the current height too:
```suggestion
                "Terminal height is too short! Toipe requires at least {} lines, got {} lines",
```

Reply:
👍🏼